src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -235,6 +237,32 @@ class Preview extends React.Component<Props> {
         return this.iframeRef.contentDocument;
     };
 
+    getPreviewWindow= (): ?Window => {
+        if (this.previewWindow) {
+            return this.previewWindow;
+        }
+
+        if (!(this.iframeRef instanceof HTMLIFrameElement)) {
+            return;
+        }
+        return this.iframeRef.contentWindow;
+    };
+
+
+    getPreviewScrollPos = ()  => {
+        const win=this.getPreviewWindow();
```

Comment:
same as above we never use any kind of abbrevations so this should be window or better `previewWindow`.